testfiles/pgfmath-gh1123.lvt

```diff
@@ -0,0 +1,33 @@
+\documentclass{minimal}
+\input{pgf-regression-test}
+
+\RequirePackage{pgfmath}
+
+\makeatletter
+\csname protected\endcsname\long\def\ASSERTPGFMATHFUNCTIONS#1#2{%
+  \pgfexpl@tl@if@eq@@ccTF{pgfmath#1@}{pgfmath#2@}{%
+    \TYPE{PASSED}%
+    \TYPE{\pgfexpl@cs@meaning@@c{pgfmath#1@}}%
+  }{%
+    \TYPE{FAILED}%
+    \TYPE{\pgfexpl@cs@meaning@@c{pgfmath#1@}}%
+    \TYPE{\pgfexpl@cs@meaning@@c{pgfmath#2@}}%
+  }%
+}
+\makeatother
+
+\START
+
+\BEGINTEST{/pgf/declare function, space(s) in arg list}
+  \pgfkeys{
+    /pgf/declare function={
+      funcA( \x,\y) = sqrt(\x^2 + \y^2);
```

Comment:
Not relevant to this case, but `\x^2` is “wrong” and should read `(\x)^2`.

Reply:
In usage `funcA(1+1,2+2)`, args are evaluated before `\pgfmathfuncA@` sees them, are they?

Reply:
Negative numbers are the problem because `\x` will be expanded before evaluation, i.e. for `\x = -2` we get `-2^2 = -4`

Reply:
true. Another trap set by operator precedence and tex's token-by-token mechanism (why another? because #1084 :_))